resolvers/policy.ts

```diff
@@ -111,29 +111,61 @@ export class PolicyStore {
     };
 }
 
-export type func_GetRelayMembers = () => Promise<undefined | EventRelayMembers | Error>;
-export const get_relay_members = (db?: DB): func_GetRelayMembers => async () => {
+export type func_GetSpaceMembers = () => Promise<SpaceMember[] | Error>;
+export type func_AddSpaceMember = (event: SpaceMember) => Promise<void | Error>;
+export type func_IsSpaceMember = (pubkey: string) => Promise<boolean | Error>;
+
+export const get_space_members = (db?: DB): func_GetSpaceMembers => async () => {
     if (!db) {
-        return new Error("get_relay_members is not supported");
+        return new Error("get_space_members is not supported");
     }
     const rows = db.query<[string]>(
         "select event from events_v2 where kind = (?)",
-        [Kind_V2.RelayMember],
+        [Kind_V2.SpaceMember],
     );
-
-    const events = [] as EventRelayMembers[];
+    const events = [] as SpaceMember[];
     for (const row of rows) {
-        const relay_member_event = parseJSON<EventRelayMembers>(row[0]);
-        if (relay_member_event instanceof Error) {
-            return relay_member_event;
+        const space_member_event = parseJSON<SpaceMember>(row[0]);
+        if (space_member_event instanceof Error) {
+            return space_member_event;
         }
-        events.push(relay_member_event);
+        events.push(space_member_event);
     }
-    if (events.length == 0) {
-        return;
-    }
-
-    const event = events.sort((e1, e2) => e1.created_at - e2.created_at)[0];
-    console.log(event);
-    return event;
+    return events;
 };
+
+export const add_space_member =
+    (args: { admin: PublicKey; db?: DB }): func_AddSpaceMember => async (event: SpaceMember) => {
```

Comment:
`db?: DB` is a temporary measure when we were still deploying to Deno Deploy. Now it's not useful.